app/controllers/api/v1/moves_controller.rb

```diff
@@ -99,6 +97,14 @@ def new_move_attributes
         )
       end
 
+      def profile_or_person_latest_profile
+        person_id = new_move_params.dig(:relationships, :person, :data, :id)
```

Comment:
I intentionally didn't add logic to check if both are passed in as this will soon go away and only set profile, to avoid adding tests and complication.

Reply:
would it make more sense to the specified profile rather than the person's latest profile if both profile_id and person_id are provided?

Reply:
spoke in person: updated this to make sure suppliers don't accidentally pass both in